src/path.jl

```diff
@@ -542,7 +542,7 @@ julia> created(p"src/FilePathsBase.jl")
 created(fp::AbstractPath) = stat(fp).ctime
 Base.isdir(fp::AbstractPath) = isdir(mode(fp))
 Base.isfile(fp::AbstractPath) = isfile(mode(fp))
-Base.islink(fp::AbstractPath) = islink(lstat(fp).mode)
+Base.islink(fp::AbstractPath) = islink(mode(fp))
 Base.issocket(fp::AbstractPath) = issocket(mode(fp))
 Base.isfifo(fp::AbstractPath) = issocket(mode(fp))
```

Comment:
Btw, should this not be
```suggestion
Base.isfifo(fp::AbstractPath) = isfifo(mode(fp))
```
?

Reply:
Yeah, I think that could also be changed.

Reply:
Changed in #185